netmiko/bintec/bintec_boss.py

```diff
@@ -0,0 +1,40 @@
+from netmiko.no_enable import NoEnable
+from netmiko.no_config import NoConfig
+from netmiko.cisco_base_connection import CiscoBaseConnection
+
+
+class BintecBossBase(NoEnable, NoConfig, CiscoBaseConnection):
+    """Support for BinTec/Funkwerk (BOSS) devices"""
+
+    def session_preparation(self) -> None:
+        self._test_channel_read(pattern=r">")
+        self.set_base_prompt()
+
+    def save_config(
+        self,
+        cmd: str = "cmd=save",
+        confirm: bool = False,
+        confirm_response: str = "",
+    ) -> str:
+        """Save Config to flash."""
+        return super().save_config(
+            cmd=cmd, confirm=confirm, confirm_response=confirm_response
+        )
+        return self._send_command_timing_str(command_string=cmd)
```

Comment:
@mnhauke Why do you need to send the `cmd=save` command twice? Once in the super() call and once in the subsequent send_command_timing() call?

Reply:
Never mind...I am just going to remove the duplicate `return` statement since it would never be executed anyways.